MFVNeutralino/plugins/TriggerEfficiency.cc

```diff
@@ -276,30 +321,87 @@ void MFVTriggerEfficiency::analyze(const edm::Event& event, const edm::EventSetu
     }
   }
 
+  if (require_metfilters) {
+    if ( ! triggerfloats->pass_metfilters ){
+      return;
+    }
+  }
```

Comment:
We should double check that this script still handles the other triggers without any issue--I think it will, since this is a configured option... But I didn't look closely enough to know 100%!

Reply:
I think it should work for other triggers. I didn't add cuts to the code. The only cuts I added were either configured or applied using a `if()` statement, which only controls whether to fill a given event into MET or METNoMu histograms but doesn't do anything else to the event.

Reply:
Oh, it's only when require_metfilters is True, of course. Thanks for checking!